src/main/java/sign_up/interface_adapters/View.java

```diff
@@ -0,0 +1,6 @@
+package sign_up.interface_adapters;
+
+public interface View {
+    public void update(boolean result, String message);
+    public void view();
```

Comment:
What is this method for exactly? IIRC this wasn't part of the observer pattern (if this is the design pattern that you're using).

Reply:
It was basically to allow me to display the original UI for signing up, but I'll look into rehauling my VM with a controller and presenter, similar to the Filter Posts use case.